tests/test_connection.c

```diff
@@ -1058,6 +1062,10 @@ void test_interface_down_retry()
   noPollCtx test_nopoll_ctx;
   pthread_t thread_a;
 
+#ifdef TEST_CONNECTION_STUCK
```

Comment:
Should this be pushed to master or is it for local test?

Reply:
It's left over from a local test. I will remove.